bufr_ncep_prepbufr_adpupa_3test.yaml

```diff
@@ -0,0 +1,264 @@
+# (C) Copyright 2020 NOAA/NWS/NCEP/EMC
+#
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+
+observations:
+  - obs space:
+      name: bufr
+
+      obsdatain: "/scratch1/NCEPDEV/da/Praveen.Kumar/JEDI/IODA_CONVR_ADPUPA_PB_2020110112/testinput/gdas.t12z.adpupa.prepbufr"
+
+      exports:
+        group_by_variable: prepbufrDataLvlCategory
+        variables:
+          # MetaData        
+          stationIdentification:
+            query: "*/SID"
+          obsTimeMinusCycleTime:
+            query: "*/DHR"
+          stationElevation:
+            query: "*/ELV"
+          prepbufrReportType:
+            query: "*/TYP"
+          dumpReportType:
+            query: "*/T29"
+          prepbufrDataLvlCategory:
+            query: "*/PRSLEVEL/CAT"
+          longitude:
+            query: "*/PRSLEVEL/DRFTINFO/XDR"
+          latitude:
+            query: "*/PRSLEVEL/DRFTINFO/YDR"
+          timeCycleProfileLevel:
+            query: "*/PRSLEVEL/DRFTINFO/HRDR"
+            
+          # ObsValue
+          dewpointTemperature:
+            query: "*/PRSLEVEL/Q___INFO/TDO"
+            transforms:
+              - offset: 273.15
+          virtualTemperature:
+            query: "*/PRSLEVEL/T___INFO/TVO"
+            transforms:
+              - offset: 273.15
+          pressure:
+            query: "*/PRSLEVEL/P___INFO/P__EVENT/POB"
+            transforms:
+              - scale: 100
+          specificHumidity:
+            query: "*/PRSLEVEL/Q___INFO/Q__EVENT/QOB"
+            transforms:
+              - scale: 0.000001
+          airTemperature:
+            query: "*/PRSLEVEL/T___INFO/T__EVENT/TOB"
+            transforms:
+              - offset: 273.15
+          heightOfObservation:
+            query: "*/PRSLEVEL/Z___INFO/Z__EVENT/ZOB"
+          windEastward:
+            query: "*/PRSLEVEL/W___INFO/W__EVENT/UOB"
+          windNorthward:
+            query: "*/PRSLEVEL/W___INFO/W__EVENT/VOB"
+                        
+          # QualityMarker
+          pressureQualityMarker:
+            query: "*/PRSLEVEL/P___INFO/P__EVENT/PQM"
+          specificHumidityQualityMarker:
+            query: "*/PRSLEVEL/Q___INFO/Q__EVENT/QQM"
+          airTemperatureQualityMarker:
+            query: "*/PRSLEVEL/T___INFO/T__EVENT/TQM"
+          heightQualityMarker:
+            query: "*/PRSLEVEL/Z___INFO/Z__EVENT/ZQM"
+          windQualityMarker:
+            query: "*/PRSLEVEL/W___INFO/W__EVENT/WQM" 
+
+          # ObsError
+          pressureError:
+            query: "*/PRSLEVEL/P___INFO/P__BACKG/POE"
+            transforms:
+              - scale: 100
+          relativeHumidityError:
+            query: "*/PRSLEVEL/Q___INFO/Q__BACKG/QOE"
+          airTemperatureError:
+            query: "*/PRSLEVEL/T___INFO/T__BACKG/TOE"
+            transforms:
+              - offset: 273.15

```

Comment:
I don't think this is necessary. The error (as far as I can tell) is the difference from the original value. Since 1K = 1C, I don't think we need a transform.

Reply:
This yaml needs to be deleted. But, I will check this for bufr_ncep_prepbufr_adpupa.yaml.

Reply:
Then it needed to be deleted for both the temperature and pressure.  Do you want me to update both?